packages/core/src/tools/create-lambda-handler.js

```diff
@@ -187,22 +177,7 @@ const loadApp = async (event, rpc, appRawOrPath) => {
 };
 
 const createLambdaHandler = (appRawOrPath) => {
-  const handler = (event, context, callback) => {
-    // Wait for all async events to complete before callback returns.
-    // This is not strictly necessary since this is the default now when
-    // using the callback; just putting it here to be explicit.
-    // In some cases, the code hangs and never exits because the event loop is not
-    // empty, so we can override the default behavior and exit after the app is done.
-    context.callbackWaitsForEmptyEventLoop = true;
-    if (event.skipWaitForAsync === true) {
-      context.callbackWaitsForEmptyEventLoop = false;
-    }
```

Comment:
Removing this since we're not using callbacks anymore, so it likely will not apply. Need to remove the corresponding z/z code and Gargoyle switch as well - [context here](https://gitlab.com/zapier/zapier/-/merge_requests/59754), [AWS docs here](https://docs.aws.amazon.com/lambda/latest/dg/nodejs-handler.html#nodejs-handler-async)

Reply:
> So it's as if context.callbackWaitsForEmptyEventLoop is now always false when we refactor code from callbacks to async/await. I assume that's fine because we don't rely on callbacks to finish anything in background. The only way to make sure this behavioral difference doesn't break existing integrations is canary testing, I think.

Yes, this is my impression as well. I _think_ it will be fine, but it's hard to know exactly how integrations will behave.

Are there any particular integrations/integration types you'd recommend to canary to make sure that this doesn't break certain functionality? I'll be canary testing integrations relying on the legacy scripting runner, and if you have ideas for other integration functionality to focus on, I'd be happy to test those cases as well.